include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -45,7 +45,12 @@ struct Global {
 };
 
 template <typename T>
-const OrtApi& Global<T>::api_ = *OrtGetApiBase()->GetApi(ORT_API_VERSION);
+const OrtApi& Global<T>::api_ =
+#ifdef EXCLUDE_REFERENCE_TO_ORT_DLL
+    *((OrtApi*)(0xDEADBEEF));
```

Comment:
So, in the customop, it can't use the ORT C APIs ?

Reply:
It can. CustomOp functions gets an OrtApi* directly, so they don't need this api_. There is no need to link CustomOp lib to the onnxruntime lib.